mirrord/cli/src/config.rs

```diff
@@ -4,7 +4,6 @@ use std::path::PathBuf;
 
 use clap::{ArgGroup, Args, Parser, Subcommand, ValueEnum};
 use clap_complete::Shell;
-use email_address::EmailAddress;
```

Comment:
Please also remove the dependency from `Cargo.toml`.

Reply:
Done